pkg/audit/manager.go

```diff
@@ -730,6 +758,9 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 	totalViolationsPerEnforcementAction map[util.EnforcementAction]int64,
 	timestamp string,
 ) error {
+	var errs []error
+	var batch []interface{}
+	enableBatch := am.pubsubSystem.IsBatchingEnabled(*auditConnection)
```

Comment:
Callers should not need to do this. They should just call `PublishBatch()`, the system sorts out the correct behavior.

pubsubSystem should not have an `IsBatchingEnabled()` method.